README.md

```diff
@@ -86,6 +86,7 @@ Start a benchmark:
     ./run_benchmark \
         --dsn postgresql://postgres@localhost/<target-db> \
         [--benchmark] <tpch|tpcds|ssb|htap> \
+	--use-server-side-cursors
```

Comment:
If this is optional, it should be in brackets. Also, we want to keep this example minimal, so unless this option is highly recommended, I would keep it out.

Reply:
removed